### PR TITLE
Expand pwd command

### PR DESCRIPTION
**Pull request by JonnyWalker81**
Expand functionality of the pwd command to better handle the different
types of shells (e.g. FilesystemShell, ValueShell, etc.).

### CODE REVIEW

File: src/commands/pwd.rs [hunk 1]
```diff
@@ -1,9 +1,9 @@
 use crate::commands::WholeStreamCommand;
+// use crate::data::{Dictionary, Value};
```
Comment (androbtech) [inline src/commands/pwd.rs]: Could you remove this comment?

File: src/commands/pwd.rs [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 use crate::errors::ShellError;
-use crate::data::{Dictionary, Value};
 use crate::parser::registry::Signature;
 use crate::prelude::*;
-use indexmap::IndexMap;
+// use indexmap::IndexMap;
```
Comment (androbtech) [inline src/commands/pwd.rs]: Same here.

File: src/commands/pwd.rs (hunk 1 above)
Comment (JonnyWalker81) [reply src/commands/pwd.rs]: Yeah, I thought I got them all, but missed a couple, I will remove them.

Comment (androbtech) [reply src/commands/pwd.rs]: Thanks!

**Pull request merged**